### PR TITLE
WD-5415 - Build kernel real-time page at /real-time

### PR DESCRIPTION
**Pull request by pedoch**
## Done

- Build the real-time kernel page (`/real-time`)

## QA

- Check out the [demo here](https://ubuntu-com-13172.demos.haus/real-time) and compare it with [the design](https://www.figma.com/file/Cr1S07Ivk632jniBAG89S8/23.10-U.com---Real-time-Ubuntu?type=design&node-id=30-1306&mode=design&t=oK05jG8BKFjcI2Vg-0) and [the copy doc](https://docs.google.com/document/d/1g9G3r4eJRPGZy5Dyz_GJhUz_iqNn-5QUhG43scnBnUA/edit).

## Issue / Card

- [WD-5415](https://warthogs.atlassian.net/browse/WD-5415)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-5415]: https://warthogs.atlassian.net/browse/WD-5415?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/real-time/index.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+<section class="p-strip">
+  <div class="row--25-75">
+    <div class="col u-hide--medium u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/454e861d-Ubuntu-real-time-Kernel-cropped.png",
+        alt="",
+        width="285",
+        height="267",
+        hi_def=True,
+        loading="auto|lazy"
```
Comment (mtruj013) [inline templates/real-time/index.html:22]:
```suggestion
        loading="auto"
```

File: templates/real-time/index.html [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    <hr class="p-rule">
+    <div class="col">
+      <h2>Getting started</h2>
+    </div>
+    <div class="col">
+      <p>The real-time kernel is available via <a href="/pro">Ubuntu Pro</a>, Canonical’s enterprise security and compliance subscription. Ubuntu Pro is free for personal and small scale commercial use in up to 5 machines.</p>
+      <p>With an Ubuntu Pro subscription, launching the real-time kernel is as easy as:</p>
+      <pre><code>pro attach &lt;token&gt;
+pro enable realtime-kernel</code></pre>
+      <a class="p-button" href="https://ubuntu.com/pro">Discover Ubuntu Pro</a>
```
Comment (mtruj013) [inline templates/real-time/index.html:149]:
```suggestion
      <a class="p-button" href="/pro">Discover Ubuntu Pro</a>
```

File: templates/real-time/index.html [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule">
+    <div class="col">
+      <h2>Run Real-time Ubuntu on Intel hardware</h2>
+    </div>
+    <div class="col">
+      <p>Optimised Real-time Ubuntu is available on 12th Gen Intel&reg; Core&trade; processors. Coupled with Intel&reg; Time Coordinated Computing (Intel&reg; TCC) and Time Sensitive Networking (IEEE TSN), the real-time kernel delivers industrial-grade performance to meet stringent latency requirements:</p>
```
Comment (mtruj013) [inline templates/real-time/index.html:161]:
```suggestion
      <p><a href="/engage/realtime-webinar-ga">Optimised Real-time Ubuntu</a> is available on 12th Gen Intel&reg; Core&trade; processors. Coupled with Intel&reg; Time Coordinated Computing (Intel&reg; TCC) and Time Sensitive Networking (IEEE TSN), the real-time kernel delivers industrial-grade performance to meet stringent latency requirements:</p>
```

File: templates/real-time/index.html [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+      <p>Intel-optimised Real-time Ubuntu is available on Intel SoCs via Ubuntu Pro:</p>
```
Comment (mtruj013) [inline templates/real-time/index.html:162]:
```suggestion
      <p>Intel-optimised Real-time Ubuntu is available on Intel SoCs via <a href="/pro/dashboard">Ubuntu Pro</a>Ubuntu Pro:</p>
```

File: templates/real-time/index.html [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+      <div class="row">
+        <div class="p-section--shallow">
+          <div class="row">
+            <div class="col-2">
+              <h3 class="p-text--small-caps">Webinars</h3>
+            </div>
+            <div class="col-4">
+              <ul class="p-list--divided u-no-margin--bottom">
+                <li class="p-list__item"><a href="https://www.brighttalk.com/webcast/6793/535454">An introduction to real-time Linux</a></li>
+                <li class="p-list__item"><a href="https://www.brighttalk.com/webcast/6793/579183?utm_source=Canonical&utm_medium=brighttalk&utm_campaign=579183">How real-time Linux accelerates industrial transformation</a></li>
```
Comment (mtruj013) [inline templates/real-time/index.html:185]: Left a question in the doc about this one, not sure if we should be including the utm or not (not a blocker though)

**Review by mtruj013 — APPROVED** [review]
Looks great, thanks @pedoch! Just some minor copy discrepancies. Also there are some text orphans here, would be good to add a break or nbsp to correct 
image

**Comment by juanruitina** [comment]
LGTM, nice work!

**Comment by lyubomir-popov** [comment]
(review not complete) low res graphic - have we provided it like that? If not, can you check if the image optimsation service parameters and ensure the resolution its 2-3x ther display size - to account for retina screens
image

spacing needs work:
- shallow stri pwith no padding bottom so we get 1.5rem at the top:
image

- u-no-margin--bottom on the h1
image

take these out of the shallow section:
image

- deep p-section wrapping the whole hero, to allow more space after the illustration:

image

**Comment by pedoch** [comment]
@lyubomir-popov ping

**Comment by lyubomir-popov** [comment]
that's the entire review @pedoch

**Comment by pedoch** [comment]
@lyubomir-popov please take another look when you can

**Comment by lyubomir-popov** [comment]
looks good - thanks @pedoch. Just one question - should we be using the [code snippet component ](https://vanillaframework.io/docs/examples/patterns/code-snippet/icon) instead of just `<pre>`?

**Review by carkod — APPROVED** [review]
LGTM

**Pull request merged**